examples/fluent-bit/grafana-loki/task-definition.json

```diff
@@ -0,0 +1,40 @@
+{
+    "family": "firelens-example-grafana-loki",
+    "taskRoleArn": "arn:aws:iam::XXXXXXXXXXXX:role/ecs_task_iam_role",
+    "executionRoleArn": "arn:aws:iam::XXXXXXXXXXXX:role/ecs_task_execution_role",
+    "containerDefinitions": [
+        {
+            "essential": true,
+            "image": "amazon/aws-for-fluent-bit:stable",
+            "name": "log_router",
+            "firelensConfiguration":{
+                "type":"fluentbit",
+                "options":{
+                    "enable-ecs-log-metadata":"true"
+                }
+            },
+            "memoryReservation": 50
+         },
+         {
+             "essential": true,
+             "image": "httpd",
+             "name": "app",
+             "logConfiguration": {
+                "logDriver":"awsfirelens",
+                "options": {
+                    "Name": "grafana-loki",
+                    "Url": "https://<userid>:<apikey>@logs-prod-us-west2.grafana.net/loki/api/v1/push",
```

Comment:
I'm confused since I see the API key here, and also in the secret value with a key called `apikey`. This means at runtime the log configuration would be:

```
"options": {
                    "Name": "grafana-loki",
                    "Url": "https://<userid>:<apikey>@logs-prod-us-west2.grafana.net/loki/api/v1/push",
                    "apikey": "value",
                    "Labels": "{job=\"firelens\"}",
                    "RemoveKeys": "container_id,ecs_task_arn",
                    "LabelKeys": "container_name,ecs_task_definition,source,ecs_cluster",
                    "LineFormat": "key_value"
                },
```

This is different than the example in the readme since now we have the `apikey` key here separate from the URL.

Reply:
Yeah make sense.

Reply:
Ok, i think there is no log configuration option "apikey": "value" so if i add it in secretOptions i don't need it in the URL? Is it kind of like a reference ?